### ISSUE TITLE
Tailwind config autodetect is failing when the tailwind import is indirect

### ISSUE THREAD

**Issue by tleunen** [sergioazoc/oxlint-tailwindcss#4]
It looks like when tailwindcss is not imported directly from the "entry point" that is detected, it then fails to find the design system.

For example, if I have a `tailwind.css` file in my package that is importing another package css which then imports tailwind, like the following:
```
@import '@company/theme/tailwind.config.css';
```

Then the autodetection fails to read the DS.

But if I manually set an `entryPoint` to that `tailwind.css` file, then it works. So the problem is really only with the auto-detection system.

@sergioazoc What do you think of automatically resolving the css imports to find the tailwind signal [in here](https://github.com/sergioazoc/oxlint-tailwindcss/blob/236c28d71068ccac89430744e5c957021b01a884/src/design-system/auto-detect.ts#L42), as well as adding `tailwind.config` as part of the candidate list?

**Comment by sergioazoc** [comment]
Fixed in v0.3.0.

The auto-detector now follows `@import` statements one level deep when the candidate CSS file doesn't contain a direct Tailwind signal. This covers both relative paths and package imports:

```css
/* src/globals.css — detected as entry point */
@import '@company/theme/tailwind.config.css';
```

```css
/* node_modules/@company/theme/tailwind.config.css */
@import 'tailwindcss';
```

The resolution walks up `node_modules` directories for package imports (monorepo-aware), and uses `path.resolve` for relative imports. Only one level is followed — no recursion — to keep auto-detection fast.

If auto-detection still doesn't find your entry point, you can always set it explicitly:

```jsonc
{
  "settings": {
    "tailwindcss": {
      "entryPoint": "tailwind.css"
    }
  }
}
```

**Comment by tleunen** [comment]
Awesome. Thank you so much for the quick response and fixes. I'll remove my own patch and try the new version shortly
